gradle-plugin/src/main/java/com/google/cloud/tools/dependencies/gradle/LinkageCheckTask.java

```diff
@@ -142,7 +143,8 @@ private boolean findLinkageErrors(Configuration configuration) throws IOExceptio
 
       ImmutableSet<LinkageProblem> linkageProblems = linkageChecker.findLinkageProblems();
 
-      LinkageProblemCauseAnnotator.annotate(classPathResult, linkageProblems);
+      ClassPathBuilder classPathBuilder = new ClassPathBuilder();
```

Comment:
Because this is classPathBuilder, I renamed the existing `classPathBuilder` to something else.